docassemble/assemblylinewizard/interview_generator.py

```diff
@@ -1148,3 +1153,231 @@ def is_person(field_name, people_vars=generator_constants.PEOPLE_VARS):
+      url=""" + repr(info['url'] if info['url'] else 'https://docassemble.org') + """,
+      packages=find_packages(),
+      namespace_packages=['docassemble'],
+      install_requires=[""" + dependencies + """],
+      zip_safe=False,
+      package_data=find_package_data(where='docassemble/""" + str(pkgname) + """/', package='docassemble.""" + str(pkgname) + """'),
+     )
+"""
+  templatereadme = """\
+# Template directory
+If you want to use templates for document assembly, put them in this directory.
+"""
+  staticreadme = """\
+# Static file directory
+If you want to make files available in the web app, put them in
+this directory.
+"""
+  sourcesreadme = """\
+# Sources directory
+This directory is used to store word translation files,
+machine learning training files, and other source files.
+"""
+  templatesreadme = """\
+# Template directory
+This directory is used to store templates.
+"""
+  # Write the standard files
+  zip_obj.writestr(os.path.join(pkg_path_prefix,"LICENSE"), licensetext)
+  zip_obj.writestr(os.path.join(pkg_path_prefix,"MANIFEST.in"), manifestin)
+  zip_obj.writestr(os.path.join(pkg_path_prefix,"README.md"), readme)
+  zip_obj.writestr(os.path.join(pkg_path_prefix,"setup.cfg"), setupcfg)
+  zip_obj.writestr(os.path.join(pkg_path_prefix,"setup.py"), setuppy)
+  zip_obj.writestr(os.path.join(pkg_path_init_prefix,"__init__.py"), initpy)
+  zip_obj.writestr(os.path.join(pkg_path_deep_prefix,"__init__.py"), ("__version__ = " + repr(info.get('version', '')) + "\n") )
+  zip_obj.writestr(os.path.join(pkg_path_questions_prefix,"README.md"), templatereadme )
+  zip_obj.writestr(os.path.join(pkg_path_sources_prefix,"README.md"), sourcesreadme )
+  zip_obj.writestr(os.path.join(pkg_path_static_prefix,"README.md"), staticreadme)
+  zip_obj.writestr(os.path.join(pkg_path_templates_prefix,"README.md"), templatesreadme)
+  
+  # Modules
+  for file in folders_and_files.get('modules',[]):
+    try:
+      zip_obj.write(file.path(),os.path.join(pkg_path_deep_prefix, file.filename))
+    except:
+      log('Unable to add file ' + repr(file))        
+  # Templates
+  for file in folders_and_files.get('templates',[]):
+    try:
+      zip_obj.write(file.path(),os.path.join(pkg_path_templates_prefix, file.filename))
+    except:
+      log('Unable to add file ' + repr(file))
+  # sources
+  for file in folders_and_files.get('sources',[]):
+    try:
+      zip_obj.write(file.path(),os.path.join(pkg_path_sources_prefix, file.filename))
+    except:
+      log('Unable to add file ' + repr(file))
+  # static
+  for file in folders_and_files.get('static',[]):
+    try:
+      zip_obj.write(file.path(),os.path.join(pkg_path_static_prefix, file.filename))
+    except:
+      log('Unable to add file ' + repr(file))
+  # questions
+  for file in folders_and_files.get('questions',[]):
+    try:
+      zip_obj.write(file.path(),os.path.join(pkg_path_questions_prefix, file.filename))
+    except:
+      log('Unable to add file ' + repr(file))
```

Comment:
```suggestion
  for folder in ['modules', 'templates', 'sources', 'static', 'questions']:
    for file in folders_and_files.get(folder,[]):
      try:
        zip_obj.write(file.path(),os.path.join(pkg_path_deep_prefix, file.filename))
      except:
        log('Unable to add file ' + repr(file))        
```

Reply:
Not quite right--the folder path isn't included?